### PR TITLE
added vCore example to create an elastic pool

### PR DESCRIPTION
**Pull request by MashaMSFT**
## Description

Added vCore as an example

## Checklist

- [ ] I have read the [_Submitting Changes_](../CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../CONTRIBUTING.md)
- [X ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/Sql/Sql/help/New-AzSqlElasticPool.md [hunk 1]
```diff
@@ -53,6 +54,32 @@ StorageMB         : 409600
...
+DatabaseCapacityMin : 0
+DatabaseCapacityMax : 2
+Family              : Gen5
+MaxSizeBytes        : 34359738368
+StorageMB           : 32768
+Tags                : 
+```
+
+
 This command creates an elastic pool in the Standard service tier named ElasticPool01. The server
```
Comment (cormacpayne) [inline src/Sql/Sql/help/New-AzSqlElasticPool.md]: @MashaMSFT you'll want to move this description to be in example 1's section and add a new description for this example that you've added to replace this.

**Comment by MashaMSFT** [comment]
@cormacpayne, thanks for the feedback! I updated the change log, though I'm not sure if I did it correctly :) and I also updated the comment.
Thanks!

**Pull request merged**